test/input/experiments/2D_arrays.gt

```diff
@@ -0,0 +1,17 @@
+element Vertex end
```

Comment:
What are the experiments? Are they part of the real test suite?

Reply:
Those are not part of the test suite. I was just trying to write few applications that use outer loop parallelization and functor. I can remove them or move them to some other folder.